src/operation/convert.ts

```diff
@@ -14,8 +14,18 @@
  * limitations under the License.
  */
 
+import { isInteger, isNaN, isNumber, toNumber } from 'lodash'
```

Comment:
this will import the whole lodash, increasing, by a lot, the bundle size.

```suggestion
import isInteger from 'lodash/isInteger'
import isNaN from 'lodash/isNaN'
import isNumber from 'lodash/isNumber'
import toNumber from 'lodash/toNumber'
```